xml/XSLT/panelfile-4-19-2.xsl

```diff
@@ -812,7 +845,7 @@ value="<xsl:value-of select="@dataString"/>"
     transition time to <xsl:value-of select="@data"/> fast seconds
   </xsl:when>
   <xsl:when test="( @type = 29 )" >
-    Control audio:
+    Control audio: 
```

Comment:
Is the space at the end of this line intentional?  If so, it might be better to use &nbsp; so that a later auto-edit doesn't remove it.

Reply:
(That should have used quotes so that that `&nbsp;` was visible)

Reply:
Can be removed without harm.